showcase/src/app/demo/slider/slider.component.html

```diff
@@ -0,0 +1,133 @@
+<example-viewer [extensions]="['HTML', 'SCSS', 'TS']" 
+				filesPath="slider/simple-horizontal-slider"
+				title="SHOWCASE.DEMO.SLIDER.SIMPLE_HORIZONTAL_SLIDER">
```

Comment:
This binding has been changed after your PR.
Could you please adapt this for `exampleTitle="SHOWCASE.DEMO.SLIDER.SIMPLE_HORIZONTAL_SLIDER"` ?